java/src/main/java/com/xxAMIDOxx/xxSTACKSxx/api/v1/menu/dto/MenuCreatedDto.java

```diff
@@ -0,0 +1,18 @@
+package com.xxAMIDOxx.xxSTACKSxx.api.v1.menu.dto;
+
+/**
+ * @author Suresh Krishnan
+ * @author ArathyKrishna
+ */
+public class MenuCreatedDto {
```

Comment:
I think this is redundant. 
There should be a generic `ResourceCreatedResponse` DTO for Menu, Category and Item creation